modopt/tests/test_opt.py

```diff
@@ -210,6 +210,10 @@ class dummy(object):
         [self.inst1.get_cost(2) for i in range(2)]
         self.inst2 = cost.costObj([dummy_inst1, dummy_inst2], cost_interval=2)
         [self.inst2.get_cost(2) for i in range(6)]
+        # Test that by default cost of False if interval is None
+        self.inst_default = cost.costObj([dummy_inst1, dummy_inst2],
```

Comment:
Perhaps change the name from `inst_default` to `inst3` or `inst_none` otherwise it's a bit confusing given that this is not the default behaviour.

Reply:
Oh yeah , codes from when it actually defaulted. Fixed it